### ISSUE TITLE
bibl with @corresp

### ISSUE THREAD

**Issue by PietroLiuzzo** [BetaMasaheft/Documentation#863]
With @thea-m we have found a quite interesting diversity of usage of the attribute corresp in bibl.

We have updated the guidelines to reflect what the intended use of this attribute is.

http://betamasaheft.eu/Guidelines/?id=bibliographic-references

there are a number of cases which you can find by XPath in oxygen or look up with this query

collection('/db/apps/BetMas/data')//t:bibl[@corresp]

in http://betamasaheft.eu/xpath

some are results of data conversion and any way needed clean up, but there are 

references to IDs of manuscripts (given plain) from BLorient481

````xml
                        <listBibl type="catalogue"> 
                           <bibl corresp="BNFabb65"> 
                           <ptr target="bm:Abbadie1859Cat"/>
                           <citedRange unit="page">75-79</citedRange>
                        </bibl>
                           <bibl corresp="UBTaeth7"> 
                           <ptr target="bm:Ewald1844Tuebingen"/>
                           <citedRange unit="page">179-180</citedRange>
                        </bibl>
                        </listBibl>

````

what does this encoding tell us? is it the bibliography of manuscripts containing the same work? what kind of information do we get from this? see also #862 

there are then plain mistakes like pointing to a xml:lang instead of xml:id. these simply need to be fixed.

see this example from LIT2290Sargis

````xml
<bibl corresp="#grc" type="BHG">
                  <ptr target="bm:BHG"/>
                  <citedRange unit="item">1322m, 1322mb</citedRange>
               </bibl>
````
 
by the way, this record does not have an <editor> which is wrong.

**Comment by PietroLiuzzo** [comment]
Where the @corresp is used to point to the id a witness listed for a work, like in LIT2317Senodo,
it works fine, but actually it is wrong against the guidelines.
````xml
<bibl corresp="blorient481">
                    <ptr target="bm:Wright1877BM"/>
                    <citedRange unit="page">2-4</citedRange>
                 </bibl>
```
should be 
````xml
<bibl corresp="#blorient481">
                    <ptr target="bm:Wright1877BM"/>
                    <citedRange unit="page">2-4</citedRange>
                 </bibl>
````

I will have to rectify the code, which will generate visualization problems until the data is also corrected.

**Comment by PietroLiuzzo** [comment]
let me correct my self. the code is smart enough to deal with this problem, and renders the `@corresp` also when it is wrongly entered (without # for the local anchor). These cases simply need to be fixed so that they are correct against the guidelines.

**Comment by eu-genia** [comment]
1) Concerning BLorient481 and other similar cases - these were one of the first mss I processed (nearly 2 years ago now) - and I tried to encode all information from the catalogue, including the bibliography provided by Wright, who, indeed, provided the bibliography on editions and on catalogue descriptions featuring the same work as far as those were available to him. At that point the idea was to preserve all that the original cataloguer had. Should it be rather deleted?

2) In one of the last discussions it has been decided that we do not have to list witnesses under a work record, these being pulled automatically - this would also mean though that we cannot assign derived IDs within the record for the `corresp ` pointer. How do we link the bibliographic record to a particular witness in that case?

**Comment by PietroLiuzzo** [comment]
nothing to delete, it is just to confirm the meaning and document it better, as well as fix correctly the wrong instances. the blorient481 encoding is fine as far as it points to the ids of witnesses as it does, but it might be unclear so the guidelines need to be more explicit about this usage. You have confirmed @thea-m interpretation so, we are all set, we only need to clarify that use in the guidelines.

your point two sounds like an entirely different matter. do you mean you have cases in the bibl/@corresp where there is BNFet125#ms_i1 (from  LIT2169Pateri)? I searched for this and found very few instances. what enrichment of information would that bring? does it translate somehow a sentence in the catalogue?
I see the point in linking bibliography to a specific part of the description or to a witness of a work, but the need to point directly to the msItem meant by the cataloguer is probably overencoding. cooccurrence of the same text in manuscripts is already supported in several search and browsing functions, this would not help that much. If that piece of bibliography only concerns BNFet125#ms_i1, rather put it there and not with the work, or if also there, without this corresp which does not add much. The user will find the bibliographic reference any way and it will not be lost in the encoding of the catalogue. or have I missed something?

I found also this example in LIT1954Mashaf, where I am not sure how to interpret. I would say, the first corresp is in LIT1954Mashaf, but then there are in the same record also #treatis2 and #treatise3.
my easiest option of a fix would be to remove LIT1954Mashaf#Treatise2

````xml
<bibl corresp="LIT1954Mashaf#Treatise2 #Treatise2 #Treatise3">
                  <ptr target="bm:Grebaut1911Mysteres"/>
               </bibl>
````

and this one in LIT1295Dersan 

````xml
<bibl corresp="#BLorient691">
                        <ptr target="bm:budge1894stmichael"/>
                        <citedRange unit="page">195-216</citedRange>
                     </bibl>
````

where the corresp needs at least to be changed to an id and that id needs to be added to the witness element.

So, again, what does this linking practice tells us? we encode the bibliography provided by the cataloguer for manuscripts containing the same work, how can we formulate this correctly in the guidelines? you see my attempt on the page linked above.

**Comment by eu-genia** [comment]
if external pointing is OK then there is probably no problem, i was confused by the remark that `<bibl corresp="blorient481">` is wrong
the example in the guidelines (correct) gives
```
<bibl corresp="#bnfet5 #bnfet146 #bdlbruce93 #bdlbruce87 #berlin395 #blor818 #blor819">
                     <ptr target="bm:Bezold1905KebraN"></ptr>
                  </bibl>
```
pointing to internal IDs assigned within the record, but if we do not list them we should be able to point to the actual IDs of the witnesses

**Comment by eu-genia** [comment]
i took the example of LIT1709Kebran from the guidelines as they are now

my question referred to what to do if there is no internal list of witnesses in the record as during the last meeting it was told to not create any internal list of witnesses for works

case 1: does that mean that i must create an internal list of witnesses after all?
case 2: i agree that in principle the bibliography on manuscripts is with manuscripts and on works with works, but above you wrote "nothing to delete", and the issue was about preserving the bibliography originally provided by the cataloguer.

**Comment by PietroLiuzzo** [comment]
I see, it is a piece of the first example, ok.

your question one: no, you do not have to create an internal list as I tried to explain in the previous comment.

your question two: as I tried to explain in the previous post, you do not have to delete anything.

The issue here is how to write down in the guidelines, in the page linked above in the best way this practice, so that we know what is possible and for what and we can document it properly. We do not have to prescribe one behaviour only, we are already supporting several, which is a feature.

If we agree on the different scenarios then I will go ahead and edit the guidelines in this way. I still think without a clear explanation of the **reason** why this encoding is needed  

The only change needed from you will be, for the wrong instances of internal ids given without  #, to add that.
